x-pack/heartbeat/monitors/browser/synthexec/enrich.go

```diff
@@ -180,14 +183,16 @@ func (je *journeyEnricher) createSummary(event *beat.Event) error {
 		down = 0
 	}
 
+	duration := je.end.Sub(je.start)
+
 	// Incase of syntax errors or incorrect runner options, the Synthetics
 	// runner would exit immediately with exitCode 1 and we do not set the duration
 	// to inform the journey never ran
 	if !je.start.IsZero() {
 		eventext.MergeEventFields(event, common.MapStr{
 			"monitor": common.MapStr{
 				"duration": common.MapStr{
-					"us": int64(je.end.Sub(je.start) / time.Microsecond),
+					"us": duration.Microseconds(),
```

Comment:
We had this guard before measuring duration before `if !je.start.IsZero() {`

Not sure if we had test for this case, could you move the duration logic back ?